man/man1/pkcstok_migrate.1.in

```diff
@@ -0,0 +1,74 @@
+.TH PKCSTOK_MIGRATE 1 "May 2020" "@PACKAGE_VERSION@" "openCryptoki"
+.SH NAME
+pkcstok_migrate \- utility to migrate a token repository to the FIPS
+compliant format introduced with openCryptoki 3.12.
+
+.SH SYNOPSIS
+\fBpkcstok_migrate\fP
+[\fB-h\fP]
+\fB--slotid\fP \fIslot-number\fP \fB--datastore\fP \fIdatastore\fP
+\fB--confdir\fP \fIconfdir\fP [ \fB--sopin\fP \fIsopin\fP \fB--userpin\fP
+\fIuserpin\fP \fB--verbose\fP \fIlevel\fP ]
+
+.SH DESCRIPTION
+This tool converts all token data, including PINs, from using PINs encrypted
+with the method of v3.11 and earlier, to being encrypted with a FIPS compliant
+method as introduced with 3.12. The new format affects the token's master key
+files (MK_SO and MK_USER), the NVTOK.DAT, and the token object files in the
+TOK_OBJ folder. 
+
+.SH Restriction:
+Only standard token formats can be migrated, which is for example not true for
+the ICSF token. 
+
+Notes:
+.br
+1. While using this tool no process using the token to be migrated must be
+   running. Especially the pkcsslotd must be stopped before running this tool.
+.br
+2. The tool creates a backup of the token repository to be migrated, and
+   performs all migration actions on this backup, leaving the original
+   repository folder completely untouched. The backup folder is located in
+   the same directory as the original repository and is named for example 
+   ../var/lib/opencryptoki/ep11tok_PKCSTOK_MIGRATE_BACKUP.
+.br
+3. After a successful migration, the original repository is renamed to
+   ../var/lib/opencryptoki/ep11tok_OLD_TO_BE_DELETED and the backup folder is
+   renamed to the original repository name, so that the migrated repository
+   can immediately be used. The old folder may be deleted by the user manually
+   later.
+.br
+4. After an unsuccessful migration, the original repository is still available
+   unchanged. 
+.br
+5. After a successful migration, the tool adds parameter 'tokversion = 3.12' to
+   the token's slot configuration in the opencryptoki.conf file. The original
+   config file is still available as opencryptoki.conf_OLD_TO_BE_DELETED and
+   may be removed by the user manually.
+
+.SH "COMMAND SUMMARY"
```

Comment:
What is the difference between COMMAND SUMMARY and OPTIONS SUMMARY? Both seem to list options.....